Simplenote/Resources/AppStoreStrings.pot

```diff
@@ -15,12 +15,14 @@ msgstr ""
 
 #. translators: App name to be displayed in the Apple App Store. Limit to 30 characters including spaces and commas!
 msgctxt "app_store_title"
-msgid "Simplenote – Notes and Todos"
+msgid "Simplenote – Notes and Todos
+"
```

Comment:
@oguzkocer This bug strikes once again 🙁 (this is what caused the notes to not get imported in GlotPress 😰)

```suggestion
msgid "Simplenote – Notes and Todos"
```

I think it was fixed in release toolkit but you might not be pointing to the version or commit having the fix yet in your `Gemfile.lock`?

Reply:
@AliSoftware It's currently pointing to the latest release: `gem 'fastlane-plugin-wpmreleasetoolkit', '~> 1.4'
` but I don't think the changes in https://github.com/wordpress-mobile/release-toolkit/pull/297 are included in that. Looking at the WPAndroid counterpart, it looks like you're planning to do a release soon. I was wondering how soon is that and whether it makes sense to wait for that or release `1.4.1` so clients will pick this fix up automatically.

Reply:
I plan to close my JP Infra project (ref: paaHJt-2sc-p2) — which includes doing a new major version of the toolkit — during this Sprint, so let's wait for that 👍

Reply:
Note that this will have to be a 2.0.0 because the changes in working on are breaking, but only if you used the app: parameter (introduced in 1.3.x) when calling the fastlane actions from your Fastfile, and I'm pretty sure only WPAndroid used those.

So technically that 2.0.0 won't be a breaking change for any other app, but will still require you to update your Pluginfile to opt-in explicitly.

So in fact feel free to do an intermediate 1.4.1 release of the toolkit just to get that fix in if you think it's more appropriate to just get that non-breaking bugfix before the next major so it gets picked up automatically (that might be a good idea after all)